boba-chain-ops/genesis/layer_two_test.go

```diff
@@ -0,0 +1,64 @@
+package genesis_test
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/bobanetwork/v3-anchorage/boba-bindings/bindings"
+	"github.com/bobanetwork/v3-anchorage/boba-bindings/predeploys"
+	"github.com/bobanetwork/v3-anchorage/boba-chain-ops/deployer"
+	"github.com/bobanetwork/v3-anchorage/boba-chain-ops/genesis"
+	"github.com/ledgerwatch/erigon-lib/common"
+	"github.com/ledgerwatch/erigon/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+var writeFile bool
+
+func init() {
+	flag.BoolVar(&writeFile, "write-file", false, "write the genesis file to disk")
+}
+
+var testKey, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
+
+func TestBuildL2DeveloperGenesis(t *testing.T) {
+	config, err := genesis.NewDeployConfig("./testdata/test-deploy-config-devnet-l1.json")
+	require.Nil(t, err)
```

Comment:
Nit: `NoError` is preferable to `Nil` when asserting no error.

Reply:
Fixed